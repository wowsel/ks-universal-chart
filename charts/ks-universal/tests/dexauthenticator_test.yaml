suite: DexAuthenticator tests
templates:
  - dexauthenticator.yaml
tests:
  - it: should create standalone DexAuthenticator
    set:
      dexAuthenticators:
        my-auth:
          applicationDomain: standalone.example.com
          sendAuthorizationHeader: true
          applicationIngressCertificateSecretName: ingress-tls
          applicationIngressClassName: nginx
    asserts:
      - matchRegex:
          path: kind
          pattern: DexAuthenticator
      - equal:
          path: apiVersion
          value: deckhouse.io/v1 