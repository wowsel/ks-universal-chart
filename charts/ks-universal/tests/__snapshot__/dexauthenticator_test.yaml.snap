should create a dex authenticator with specified configuration:
  1: |
    - developers
    - admins
  2: |
    - domain: app1.example.com
      whitelistSourceRanges:
        - 10.0.0.0/8
    - domain: app2.example.com
